lib/puppet/parser/functions/hiera_include.rb

```diff
@@ -89,7 +89,7 @@ module Puppet::Parser::Functions
 result must be post processed to get exactly the same result, for example using simple hash/array `+` or
 with calls to stdlib's `deep_merge` function depending on kind of hiera call and setting of merge in hiera.yaml.
 
-See [the documentation](http://links.puppet.com/hierainclude) for more information
+See [the documentation](https://puppet.com/docs/puppet/latest/hiera_intro.html#assigning-classes-to-nodes-with-hiera-hierainclude) for more information
```

Comment:
This anchor doesn't seem to exist on the page?

Reply:
We probably shouldn't link to deprecated features' docs anyway, so I'll replace with a link to the Hiera-to-lookup migration guide.